libs/langchain/langchain/llms/databricks.py

```diff
@@ -424,16 +455,16 @@ def _call(
         # TODO: support callbacks
 
         request = {"prompt": prompt, "stop": stop}
+        if self._client.llm:
+            request.update(self._llm_params)
+            request.update(self.model_kwargs or self.extra_params)
+        else:
+            request.update(self.model_kwargs or self.extra_params)
         request.update(kwargs)
-        if self._params:
-            request.update(self._params)
+        if "stop" in request and not request["stop"]:
+            del request["stop"]
```

Comment:
We hit this error with an endpoint serving Anthropic:

```
{"error_code":"INVALID_PARAMETER_VALUE","message":"INVALID_PARAMETER_VALUE: stop was not of type List[String]: null"}
```

This change is to avoid this error.